packages/tools/devtools/devtools-view/src/components/ContainerRuntimeView.tsx

```diff
@@ -0,0 +1,94 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { Tree as FluentTree } from "@fluentui/react-components";
+import {
+	GetRootDataVisualizations,
+	type HasContainerKey,
+	type ISourcedDevtoolsMessage,
+	type InboundHandlers,
+	RootDataVisualizations,
+	type RootHandleNode,
+	handleIncomingMessage,
+} from "@fluidframework/devtools-core/internal";
+import React from "react";
+
+import { useMessageRelay } from "../MessageRelayContext.js";
+
+import { Waiting } from "./Waiting.js";
+import { TreeDataView } from "./data-visualization/index.js";
+
+const loggingContext = "INLINE(VIEW)";
+
+/**
+ * {@link ContainerRuntimeView} input props.
+ */
+export type ContainerRuntimeViewProps = HasContainerKey;
+
+/**
+ * Displays the data inside a container.
+ *
+ * @remarks
+ *
+ * Dispatches container runtime rendering based on those provided view {@link TreeDataView}.
+ */
+export function ContainerRuntimeView(props: ContainerRuntimeViewProps): React.ReactElement {
```

Comment:
Nit: Since this is still about Container _data_, I would probably recommend calling it `ContainerDataView` rather than `ContainerRuntimeView`.